backend/src/main/java/cz/muni/fi/pa165/pneuservis/backend/entity/Customer.java

```diff
@@ -44,13 +44,17 @@
     @NotNull
     private String phoneNumber;
 
+    private boolean vip;
+
     @ElementCollection
     private Set<VehicleTypeEnum> typesOfCar;
 
     @OneToMany(mappedBy = "customer")
     private Set<Order> orders;
 
-    public Customer(){}

```

Comment:
There should stay a constructor that doesn't take any parameters - for Spring purposes if I'm not mistaken.